contracts/sg-eth-airdrop/src/helpers/constants.rs

```diff
@@ -0,0 +1,11 @@
+pub const GENERIC_WHITELIST_LABEL: &str = "Generic Whitelist for Airdrop";
+pub const NATIVE_DENOM: &str = "ustars";
+pub const CONTRACT_NAME: &str = "crates.io:sg-eth-airdrop";
```

Comment:
Weird to see something as critical as the contract name hidden inside a folder called "helpers", out of context from where it is set.